greymatter-io/gitops-plus
# update dir structure and package naming to match gitops-core schema

[sc-19626]

To test start from the main repo
```
cue eval -c EXPORT.cue --out yaml -e configs > main-output.txt
git checkout update-scaffolding
cue eval -c EXPORT.cue --out yaml -e configs > update-output.txt
diff main-output.txt update-output.txt
```

These two should match.
